### PR TITLE
GROOVY-7946 - allow writable values for StreamingJsonBuilder

### PR DESCRIPTION
**Pull request by graemerocher**
See https://issues.apache.org/jira/browse/GROOVY-7946

### CODE REVIEW

**Comment by shils** [comment]
+1

File: subprojects/groovy-json/src/main/java/groovy/json/StreamingJsonBuilder.java [hunk 1]
```diff
@@ -651,6 +655,18 @@ public void call(String name, JsonOutput.JsonUnescaped json) throws IOException
             writer.write(json.toString());
         }
 
+        /**
+         * Writes the given Writable as the value of the given attribute name
+         *
+         * @param name The attribute name The attribute name
+         * @param json The value The writable
```
Comment (jwagenleitner) [inline subprojects/groovy-json/src/main/java/groovy/json/StreamingJsonBuilder.java]: descriptions for the params seems to be repeated twice.

**Comment by jwagenleitner** [comment]
+1

File: subprojects/groovy-json/src/main/java/groovy/json/StreamingJsonBuilder.java (hunk 1 above)
Comment (graemerocher) [inline subprojects/groovy-json/src/main/java/groovy/json/StreamingJsonBuilder.java]: Nice catch. Fixed.

**Comment by graemerocher** [comment]
Any chance this can be merged? Thanks...

**Comment by graemerocher** [comment]
Actually hold off merging, found an a potential issue and will revise the test case and update the pull request

**Comment by graemerocher** [comment]
Revised pull request, since it created issues handling `GString` instances which are also `Writable`

**Comment by graemerocher** [comment]
The failures on Travis seem to be unrelated network issues?

**Comment by jwagenleitner** [comment]
PR merged, thanks!  Travis doesn't seem to always like building against the 2_4_X branch, seems to fail spuriously.  Hope you don't mind but I squashed the commits.

**Pull request closed without merging**